src/pages/ExploreIngredients/ExploreIngredients.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { BottomMenu, Header, Loading } from '../../components';
+import { getIngredientsList, getURLIngredientsImg } from '../../services/api';
+
+const ExploreIngredients = ({ match: { path } }) => {
+  const [loading, setLoading] = useState(true);
+  const [ingredientsName, setIngredientsName] = useState([]);
+  const page = path.includes('comidas');
+  const pathIngredient = page ? 'comidas' : 'bebidas';
+  const type = page ? 'meal' : 'cocktail';
+  const ingredientNameKey = page ? 'strIngredient' : 'strIngredient1';
+
+  useEffect(() => {
+    getIngredientsList(type).then((resp) => {
+      setIngredientsName(resp.slice(0, 12));
+      setLoading(false);
+    });
+  }, []);
+
+  if (loading) return <Loading />;
+  return (
+    <div>
+      <Header pageTitle="Explorar Ingredientes" />
+      {ingredientsName.map((ingredient, index) => (
+        <Link
+          to={{ pathname: `/${pathIngredient}`, searchByIngredient: ingredient[ingredientNameKey] }}
```

Comment:
```suggestion
          to={{ pathname: `/${pathIngredient}`, ingredient: ingredient[ingredientNameKey] }}
```

Acho que não é uma boa o nome da key ser um verbo.